### ISSUE TITLE
Link Color Contrast is Too Low

### ISSUE THREAD

**Issue by Dana94** [dwyl/dwyl-site#479]
Referencing issue #448 
Some of the links don't have a high enough color contrast ratio for accessibility.

![66085867-6b289480-e540-11e9-9bfe-92305f22feb4](https://user-images.githubusercontent.com/20132264/66172979-02164f00-e61c-11e9-94a8-c9bd299f9e32.png)

**Comment by stroudn1** [comment]
Hi @Dana94 ! Are you looking for what links have low contrast or for fixes? What specific colors would you be looking for? I'd be interested in looking into this :)

**Comment by iteles** [comment]
@Dana94 @stroudn1 Which colour hex would you propose as an alternative?
I'm assuming all links on the site are the same colour but it's worth double-checking this in the CSS & HTML too!

**Comment by Dana94** [comment]
@stroudn1 Great, I'll assign this to you. 😃 

@iteles I haven't had a chance to go through the whole site yet but it seems the color contrast issues are primarily on the white sections of the site. For example, the links on the [portfolio page](https://dwyl.com/portfolio). 

![image](https://user-images.githubusercontent.com/20132264/66307532-ac1f0100-e8d2-11e9-94fc-d6a914ba15f9.png)

I would suggest #00846a to replace the `.dwyl-mint` color but it does look different from dwyl's brand color so that's up to your decision if that would be acceptable.

![image](https://user-images.githubusercontent.com/20132264/66307752-42532700-e8d3-11e9-8e53-c06eb327f2ad.png)

**Comment by stroudn1** [comment]
Hi @Dana94 ! I just wanted to let you know I'm ready to make the pull request but I seem to be getting an error when I try to run `git commit -m`. I'm still trying to debug my issue.
![image](https://user-images.githubusercontent.com/17433156/66625307-b9353c00-ebc1-11e9-815d-a45e34be21c4.png)

**Comment by iteles** [comment]
@stroudn1 It looks like @codybarr has submitted a PR for this already!

![image](https://user-images.githubusercontent.com/4185328/66641546-e100c000-ec12-11e9-8345-292c7f288b46.png)

I haven't had a chance to review it yet but if you're able to take a look and add review comments (let  us know if you need some help with that) maybe that would be a good first step?

**Comment by Dana94** [comment]
@stroudn1 Try this:

`> git commit -n`
You should see something like this if using Git bash, mine is Nano.
![image](https://user-images.githubusercontent.com/20132264/66687794-f517e200-ec51-11e9-8af2-d34cf8434e3a.png)
Enter in your message, it shows in yellow in the image.
![image](https://user-images.githubusercontent.com/20132264/66687805-006b0d80-ec52-11e9-89e3-a84589ad818f.png)
Ctrl + x to save
![image](https://user-images.githubusercontent.com/20132264/66687821-0c56cf80-ec52-11e9-8433-f249d240609b.png)
Type y to confirm, then hit enter.
![image](https://user-images.githubusercontent.com/20132264/66687833-18db2800-ec52-11e9-9900-c68fde3cc4ea.png)
`> git log` to see the commit message you made.

Hope this helps!

**Comment by codybarr** [comment]
Sorry @stroudn1 saw this was still up for grabs. I should have checked with you first to see if you had already started work on this.

**Comment by stroudn1** [comment]
@codybarr no worries! I'm still learning/trying to fix my error. I'm going to see if I can fix it and send a PR too if that's cool with you and @Dana94 

Thank you for the steps @Dana94 ! I'm going to give this a try. And thank you as well @iteles
